admin/src/test/java/com/maple/admin/service/DefaultBossItemAdminServiceTest.java

```diff
@@ -54,7 +54,7 @@ void setUp() {
     }
 
     @Test
-    void 고정_보스_아이템_dto가_null_이면_생성_실패() {
+    void 고정_보스_아이템_생성_실패__고정_보스_아이템_dto가_null() {
```

Comment:
```suggestion
    void 고정_보스_아이템_생성_실패__dto가_null() {
```

`고정_보스_아이템` 이게 중복되는거 같습니다.